app/components/Browser/Browser.jsx

```diff
@@ -50,6 +50,15 @@ class Browser extends Component
         super( props );
         this.state = {};
     }
+    componentWillMount()
```

Comment:
Cosmetic thing, perhaps keep an empty line to separate from previous function's body. Also perhaps remove the first empty line in this function's body.
```suggestion

    componentWillMount()
```